sway-launcher-desktop.sh

```diff
@@ -256,9 +256,13 @@ for PROVIDER_NAME in "${!PROVIDERS[@]}"; do
 done
 
 COMMAND_STR=$(
-  fzf +s -x -d '\034' --nth ..3 --with-nth 3 \
+  fzf --ansi +s -x -d '\034' --nth ..3 --with-nth 3 \
     --preview "$0 describe {2} {1}" \
-    --preview-window=up:3:wrap --ansi \
+    --preview-window=up:2:noborder \
+    --no-mouse --no-multi --cycle \
```

Comment:
Please keep mouse support enabled unless there's a very good reason to disable it

Reply:
That makes sense, I assumed people using this wouldn't want the mouse anyway; there's no explicit need to disable it.

I'll go and add the glyph stuff to the config file then (tomorrow at the earliest, though). What's your take on color customization?

I'm not a shell scripting wiz so the first thing that'd come to mind implementation wise would be defining overridable vars for each color option that `fzf` allows customization of (there's 16 of them) and then doing the same thing I'd do for the glyphs, config-file-wise. Perhaps chuck them into an array so their names can be re-used and iterated over for both config file and env var parsing?

And yeah, the amount of stuff you can do with `fzf` is staggering. It's kind of funny that there's pretty much a complete ecosystem around it, I've not heard of it before bumping into your launcher.

Reply:
I'd suggest you open a new PR for the customization stuff since this one is pretty much RTM except for the mouse support.

Your approach sounds reasonable. Part of me wants to just add a `FZF_COLORS` variable that contains the full string as documented [here](https://github.com/junegunn/fzf/wiki/Color-schemes), but let's see what you come up with. If I read correctly, a default value of `-1` would retain the current behaviour of simply using the current terminal color scheme, right?

Reply:
Sounds good. I'll remove the mouse bit then and push that. The manpage says `Ansi color code of -1 denotes terminal default foreground/background color.` So passing that as default for everything would return a 2-color launcher, unless I'm missing something.

Letting people pass something like `FZF_COLORS` would be a nice easy fix, but looking at the expected format that'd get really long really quick. I think I'll go figure out how to work some shell magic with substitutions and expansions for that, it's about time I get a more thorough grasp of those.

Cheers